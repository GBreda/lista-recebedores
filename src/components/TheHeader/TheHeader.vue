<template>
  <header class="the-header">
    <img data-test-id="logo" src="/src/assets/images/logo.png" alt="Transfeera logo" />
  </header>
</template>

<style lang="scss" scoped>
.the-header {
  padding: 1rem 0 1.5rem 5rem;
}
</style>
