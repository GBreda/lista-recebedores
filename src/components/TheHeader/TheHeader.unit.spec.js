import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import TheHeader from './TheHeader.vue'

describe('TheHeader', () => {
  let wrapper

  it('should mount the component', () => {
    wrapper = mount(TheHeader)

    expect(wrapper.vm).toBeDefined()
  })

  it('should match the snapshot', () => {
    wrapper = mount(TheHeader)

    expect(wrapper.element).toMatchSnapshot()
  })

  it('should contain alt text on the logo', () => {
    wrapper = mount(TheHeader)

    const logo = wrapper.find("[data-test-id='logo']")

    expect(logo.attributes('alt')).toBeTruthy()
    expect(logo.attributes('alt')).toContain('Transfeera logo')
  })
})
