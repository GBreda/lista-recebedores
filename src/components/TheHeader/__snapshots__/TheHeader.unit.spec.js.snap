// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TheHeader > should match the snapshot 1`] = `
<header
  class="the-header"
  data-v-27be5419=""
>
  <img
    alt="Transfeera logo"
    data-test-id="logo"
    data-v-27be5419=""
    src="/src/assets/images/logo.png"
  />
</header>
`;
